src/components/Trades/AddTradeLineItem.js

```diff
@@ -0,0 +1,169 @@
+import { useState, useEffect } from "react";
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+
+// Firebase
+import { push, ref, set } from "firebase/database";
+import { database } from "../../firebase.js";
+import { auth } from "../../firebase.js";
+
+// CSS
+import "./AddTradeLineItem.css";
+import { propTypes } from "react-bootstrap/esm/Image.js";
+
+const TRADES_KEY = "trades";
+
+const AddTradeLineItem = (props) => {
+  const [user, setUser] = useState("noUser");
+  const [userDisplayName, setUserDisplayName] = useState("noUserDisplayName");
+  const [nameField, setNameField] = useState("");
+  const [codeField, setCodeField] = useState("");
+  const [buySellField, setBuySellField] = useState("Buy");
+  const [dateField, setDateField] = useState("");
+  const [priceField, setPriceField] = useState("");
+  const [currencyField, setCurrencyField] = useState("");
+  const [platformField, setPlatformField] = useState("");
```

Comment:
Can probably use a single state variable for this. `formFields` or something like that, which is an object. Way easier to maintain the component then